docs/releases.rst

```diff
@@ -5,6 +5,17 @@
 ======================
 
 
+The :ref:`/plugins/report/reportportal` plugin now supports
+importing of tmt subresults as child test items.
+
+The :ref:`/plugins/report/reportportal` plugin now handles the
+timestamps for ``custom`` and ``restraint`` results correctly. It
+should never happen that the result's ``end-time`` will be higher
+then the ``start-time``. It should be also ensured that the start
+time of all launch items is the same or higher then the start time
+of a parent item/launch.
```

Comment:
is this correct? you have `should never happen...end time...be higher than the start time`?

IMO it should be
```suggestion
should never happen that the result's ``start-time`` will be higher
than the ``end-time``. It should be also ensured that the end
time of all launch items is the same or higher than the start time
of a parent item/launch.
```

Reply:
Thanks! Fixed in "child" PR here: https://github.com/teemtee/tmt/pull/3356/commits/eb7b34aba30d749ac91ddf0c97ce2557452c46c5

This PR is based on top of `feature/reportportal-improvements` (https://github.com/teemtee/tmt/pull/3356) and still needs a rebase. I plan to do it as soon as we resolve all the issues there.